src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAOStub.java

```diff
@@ -2,15 +2,16 @@
 
 import com.tvtracker.enterprise.dto.MediaEntry;
 import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
 
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 
-@Component
+@Repository
```

Comment:
Changed @Component to @Repository because this file is dealing with persistence and should be annotated as such.
https://www.baeldung.com/spring-component-repository-service

Reply:
Good.